elliottlib/rpm_utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import Dict, Optional
```

Comment:
Might be nice with comment with a link to the doozer implementation would be helpful